core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -334,7 +416,8 @@ export class BaseUser {
    * This function navigates to the given URL.
    */
   async goto(url: string): Promise<void> {
-    await this.page.goto(url, {waitUntil: 'networkidle0'});
+    await this.page.goto(url);
+    await this.page.waitForNavigation({waitUntil: ['networkidle2', 'load']});
```

Comment:
You can combine the line with the one above by calling the waitUntil options inside this.page.goto()

Reply:
Done.